src/db/db.ts

```diff
@@ -0,0 +1,23 @@
+import { Sequelize } from "sequelize";
+import dotenv from "dotenv";
+dotenv.config();
+
+const sequelize = new Sequelize({
+	dialect: "postgres",
+	host: process.env.DB_HOST,
+	port: Number(process.env.DB_PORT),
```

Comment:
A simpler way to cast a string to a number is +.

```typescript
+process.env.DB_PORT
```

Reply:
noted